crates/biome_js_analyze/src/react.rs

```diff
@@ -132,6 +132,13 @@ impl ReactLibrary {
             Self::ReactDOM => "ReactDOM",
         }
     }
+
+    pub const fn get_import_aliases(self) -> &'static [&'static str] {
+        match self {
+            Self::React => &["@rbxts/react"],
```

Comment:
Why did we not add this to `import_names()` itself? It already provides a few aliases there, so the imports work with both React and Preact. I don't mind adding an extra one.